src/components/Tree/helpers/getNextItemToFocus.ts

```diff
@@ -1,13 +1,14 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
+import { UniqueIdentifier } from '@dnd-kit/core';
 import type { TreeItemState, TreeState } from '../types';
 
 export const getNextItemToFocus = (
-    id: string,
-    currentItemState: TreeItemState,
+    id: UniqueIdentifier,
+    currentItemState: TreeItemState | undefined,
```

Comment:
```suggestion
    currentItemState: TreeItemState,
```
should never be `undefined`

Reply:
Maybe logically it can't but technically it can be undefined as it is an item state, anyway this way we reduce cognitive complexity and lose nothing as functions were returning undefined even before this change. In some future iterations maybe we can do a proper clean code refactor.